manifests/fpm/pool.pp

```diff
@@ -156,7 +156,10 @@
   $base_dir                  = undef,
 ) {
 
-  include ::php::params
```

Comment:
It's a bit late to `include ::php::params` here.  If it's not already been included, the type default for `root_group` would be missing.